stubs/app/Models/ConnectedAccount.php

```diff
@@ -41,4 +41,13 @@ class ConnectedAccount extends SocialstreamConnectedAccount
         'updated' => ConnectedAccountUpdated::class,
         'deleted' => ConnectedAccountDeleted::class,
     ];
+
+    public function jsonSerialize()
```

Comment:
@rokde Is there a reason you're using `jsonSerialize`? I agree with the change of responsibility here, but I think this method is the wrong one to use. The same applies for `toArray` and `toJson`. 

My reasoning is that the above methods are responsible for quite a lot when returning models from the controller (for instance, in a `ResourceCollection`). This change would effectively be a breaking one for developers relying on that functionality. 

Could you also move the method to the abstract? Having it in the stub means that developers who have already published their models won't inherit this unless they republish their assets. Resulting in the middleware returning all unguarded attributes.